modules/builtin/src/backend/variables/complex.ts

```diff
@@ -1,7 +1,7 @@
 import { BoxedVariable, PrimitiveVarType } from 'botpress/sdk'
 import { BaseVariable } from 'common/variables'
 
-import common from './common'
+import { common } from './common'
```

Comment:
If you want to export more stuff from './common', please rename "common", what is it exactly ?